### ISSUE TITLE
Emit json with proper types.

### ISSUE THREAD

**Issue by SwooshyCueb** [irods/irods_rule_engine_plugin_audit_amqp#103]
It appears that the json emitted by this plugin is not typed properly, encoding several fields as strings that should be integers and whatnot.
In our elk stack container, we have to convert several string fields to integers to get the kibana dashboard working, and it appears to me that these fields should already be integers.

**Comment by SwooshyCueb** [comment]
See irods/contrib#26 for more context

**Comment by korydraughn** [comment]
@SwooshyCueb Is this complete?

**Comment by SwooshyCueb** [comment]
I don't think so. It looks like we're using `irods::re_serialization::serialize_parameter`, which, I believe, serializes everything to string:string maps:
https://github.com/irods/irods_rule_engine_plugin_audit_amqp/blob/7cf87a10ec6c0d74f88fbc42c793831b3b233218/src/main.cpp#L427-L429

**Comment by korydraughn** [comment]
That's correct. That is used to make some data types available to the NREP.
